hmda-analytics/src/main/scala/hmda/analytics/query/LarComponent.scala

```diff
@@ -12,15 +12,15 @@ trait LarComponent {
   class LarRepository(tableName: String, config: DatabaseConfig[JdbcProfile]) {
     def deleteByLei(lei: String): Future[Int] =
       config.db.run {
-        sqlu"DELETE FROM #${tableName} WHERE lei = ${lei}"
+        sqlu"DELETE FROM #${tableName} WHERE UPPER(lei) = ${lei.toUpperCase}"
       }
 
     def insert(le: LarEntity): Future[Int] =
       config.db.run {
         sqlu"""INSERT INTO #${tableName}
         VALUES (
           ${le.id},
-          ${le.lei},
+          ${le.lei.toUpperCase},
```

Comment:
I don't think that we should modify the institution's LEI in our db. The other line change should be sufficient to make sure we purge our db of old records regardless of LEI casing.

Reply:
Not modifying the LEI to upper case could cause mismatches down the stream while doing data analysis. The data analysts will have to write `UPPER` in each query.